src/main.rs

```diff
@@ -41,6 +53,25 @@ enum Commands {
         /// If this path is provided and it exists, the private key is read from this file and used, if it does not exist the private key will be persisted to this location.
         #[clap(long)]
         key: Option<PathBuf>,
+        /// Optional rpc port, defaults to 4919
+        #[clap(long)]
+        rpc_port: Option<u16>,
```

Comment:
should be a "global" option, like `keylog`?

Reply:
It only applies to some ops, so no. I guess there might be some clap way to DRY this, but I think it is fine to keep it as a separate option.